### PR TITLE
Arrays last part

### PR DESCRIPTION
**Pull request by busilapetrica**

### CODE REVIEW

File: src/main/java/org/example/C05_P44_Repeating_Elements_In_Array.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+/******************************************************************************
+ 44.Write a program in java to find the two repeating elements in a given array.
+ Expected Output :
+ The given array is : 2 7 4 7 8 3 4
+ The repeating elements are: 7 0
+ *******************************************************************************/
+public class C05_P44_Repeating_Elements_In_Array {
+    private C05_P44_Repeating_Elements_In_Array () {}
+    public static String findAndPrintRepeatingElements(int[] array) {
+        Arrays.sort(array);
```
Comment (andreiursudev) [inline src/main/java/org/example/C05_P44_Repeating_Elements_In_Array.java:12]: Reimplement this problem without using sort.

File: src/main/java/org/example/C05_P50_Spiral_Matrix.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+ 6  7  8  9  10
+ 11 12 13 14 15
+ 16 17 18 19 20
+ The spiral form of above matrix is:
+ 1 2 3 4 5 10 15 20 19 18 17 16 11 6 7 8 9 14 13 12
+ *******************************************************************************/
+public class C05_P50_Spiral_Matrix {
+    private C05_P50_Spiral_Matrix () {}
+    public static String makeArraySpiral(int[][] matrix){
+        int n=4;
```
Comment (andreiursudev) [inline src/main/java/org/example/C05_P50_Spiral_Matrix.java:16]: Don't use hardcoded values. n should be dynamic in order for your solution to work for all matrix sizes.

File: src/main/java/org/example/C05_P30_Matrix_Equals.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ The second matrix is :
+ 1 2
+ 3 4
+ The Matrices can be compared :
+ Two matrices are equal.
+ *******************************************************************************/
+public class C05_P30_Matrix_Equals {
+    private C05_P30_Matrix_Equals() {}
+
+    public static boolean checkEgualMatrix(int[][] matrix1, int[][] matrix2){
```
Comment (andreiursudev) [inline src/main/java/org/example/C05_P30_Matrix_Equals.java:30]: Typo. Equal not Egual

File: src/main/java/org/example/C05_P44_Repeating_Elements_In_Array.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package org.example;
+import java.util.Arrays;
+/******************************************************************************
+ 44.Write a program in java to find the two repeating elements in a given array.
+ Expected Output :
+ The given array is : 2 7 4 7 8 3 4
+ The repeating elements are: 7 0
```
Comment (andreiursudev) [inline src/main/java/org/example/C05_P44_Repeating_Elements_In_Array.java:7]: 0 is not a repeating element.

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/C05_P44_Repeating_Elements_In_Array.java (hunk 4 above)
Comment (busilapetrica) [reply src/main/java/org/example/C05_P44_Repeating_Elements_In_Array.java]: ok. I changed... description of the problem was not good. the repeating element must be 4 and 7.. and test is checking correctly

**Pull request merged**